src/deck/drivers/src/locodeck.c

```diff
@@ -580,7 +580,13 @@ static const DeckDriver dwm1000_deck = {
   .pid = 0x06,
   .name = "bcDWM1000",
 
-  .usedGpio = 0,  // FIXME: set the used pins
+#ifdef LOCODEC_USE_ALT_PINS
+  .usedGpio = DECK_USING_IO_1 | DECK_USING_IO_2 | DECK_USING_IO_3,
+#else
+   // (PC10/PC11 is UART1 TX/RX)
+  .usedGpio = DECK_USING_IO_1 | DECK_USING_PC10 | DECK_USING_PC11,
+#endif
+  .usedPeriph = DECK_USING_SPI,
```

Comment:
from what I remember, this also uses timers?

Reply:
I cannot find any calls to TIM_functions ... you got any pointers on where to look?